python/ray/air/util/data_batch_conversion.py

```diff
@@ -217,10 +217,10 @@ def _convert_batch_type_to_numpy(
                 )
         return data
     elif pyarrow is not None and isinstance(data, pyarrow.Table):
-        from ray.air.util.tensor_extensions.arrow import (
+        from ray.data._internal.arrow_ops import transform_pyarrow
```

Comment:
Why is this still importing from data? Should this util also be moved to data?

Reply:
oh, the reason this is here is because this utility is still dependent from ray.train.predictors, so this is a duplicate. How about let's just address this in a later PR when we remove predictors?